response/page_path.go

```diff
@@ -2,13 +2,161 @@ package response
 
 import (
 	"context"
+	"net/url"
+	"regexp"
+	"strings"
 
 	"github.com/ONSdigital/log.go/v2/log"
 )
 
-func getPagePath(ctx context.Context, uri string) string {
+var resourceEndpoints = []string{"/chartconfig", "/chartimage", "/embed", "/chart", "/resource", "/generator", "/file", "/export"}
+
+var visualisationsEndpointRegexp = regexp.MustCompile(`(^/visualisations/.+?)/`)
+var bulletinsOrArticlesPathRegexp = regexp.MustCompile(`(.+/(bulletin|article)s(?:/[^/]+){2})`)
+var methodologiesOrQMIsOrAdHocsPathRegexp = regexp.MustCompile(`.+/(methodologie|qmi|adhoc)s/([^/]+)`)
```

Comment:
```suggestion
var bulletinsOrArticlesPathRegexp = regexp.MustCompile(`(.+/(bulletins|articles)(?:/[^/]+){2})`)
var methodologiesOrQMIsOrAdHocsPathRegexp = regexp.MustCompile(`.+/(methodologies|qmis|adhocs)/([^/]+)`)
```

make these more clear/searchable (i.e. if we want to search our code for `methodologies`, this makes it easier)